src/index.js

```diff
@@ -97,50 +97,57 @@ function flags () {
     tokenContract: config.tokenContract,
     communityContract: config.communityContract
   }
-  devLog('flags', flags_)
   return flags_
 }
 
 // Start elm app with flags
 const app = Elm.Main.init({
   flags: flags()
 })
+Sentry.addBreadcrumb({
+  message: 'Started Elm app',
+  level: 'info',
```

Comment:
Since Sentry has `Severity` "enum" for the level I think it's better to use `Sentry.Severity.Info` and other "variants" all over the place. Now there are several `'info'` and several `Sentry.Severity.Info`.

```suggestion
  level: Sentry.Severity.Info,
```